lib/task/import/csvImportBaseTask.class.php

```diff
@@ -238,6 +238,37 @@ static function importEvents(&$import)
       // Add row culture to fetch place term in event creation/update
       $eventData['culture'] = $import->columnValue('culture');
 
+      // If in update mode and event's a creation event with an existing actor
+      // specified, see if the event seems like it should be updated
+      // rather than created.
+      if ($eventTypeId == QubitTerm::CREATION_ID && $import->matchAndUpdate
```

Comment:
I think we should look for all event types, not just creation events. I'd remove that condition and add the type to the query directly.